src/Transport/GrpcTransport.php

```diff
@@ -188,6 +188,21 @@ public function startBidiStreamingCall(Call $call, array $options)
             $call->getDescriptor(),
             $this->logger
         );
+
+        if ($this->logger) {
+            $requestEvent = new RpcLogEvent();
+
+            $requestEvent->headers = $options['headers'];
```

Comment:
Shouldn't this also be 
```suggestion
            $requestEvent->headers = $options['headers'] ?? [];
```

I see that on line 185 it's checking if the key is set